src/v2/guide/syntax.md

```diff
@@ -152,6 +152,18 @@ The `v-` prefix serves as a visual cue for identifying Vue-specific attributes i
 <a :href="url"> ... </a>
 ```
 
+> New in 2.6
+
+In 2.6 a separate shorthand for explicit DOM property bindings (with the `.prop` modifier) have been introduced:
+
+``` html
+<!-- full syntax -->
+<video v-bind:muted.prop="isMuted"> ... </video>
+
+<!-- shorthand -->
+<video .muted="isMuted"> ... </video>
```

Comment:
TBH I think this shorthand somewhat breaks the existing unified syntax of Vue's directives:

```
v-directive:argument.modifier="value"
```

Where `:` always follows with an argument and `.` always follows with a modifier. `.muted` along may look like a modifier without directive/argument part and might be somewhat confusing if users haven't got to this part of the documentation yet.

Reply:
A modifier would only be a modifier if it has something to modify, right? So a standalone modifier doesn't even make sense.